octobot/backtesting/octobot_backtesting_factory.py

```diff
@@ -0,0 +1,36 @@
+#  Drakkar-Software OctoBot
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+from octobot_backtesting.api.backtesting import get_backtesting_data_files
+
+from octobot.api.backtesting import create_independent_backtesting, initialize_and_run_independent_backtesting, \
+    join_independent_backtesting, stop_independent_backtesting
+from octobot.octobot import OctoBot
+
+
+class OctoBotBacktestingFactory(OctoBot):
```

Comment:
Good idea !
I think this class should not be named ```OctoBotBacktestingFactory``` though because it's not a factory, it's an OctoBot. What about ```BacktestingOctoBot ``` ?

Reply:
I that was my first idea but I don't like the similarity with OctoBotBacktesting.
We can rename OctoBotBacktesting later and then rename OctoBotBacktestingFactory what do you think ?